kratos/python/add_bounding_box_to_python.cpp

```diff
@@ -0,0 +1,63 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Riccardo Rossi
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "python/add_bounding_box_to_python.h"
+#include "geometries/bounding_box.h"
```

Comment:
Where is the OBB I added some time ago?

We may duplicated code...

Reply:
this https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/includes/oriented_bounding_box.h?

it seems that `BoundingBox` is previous to `OrientedBoundingBox`

Reply:
https://github.com/KratosMultiphysics/Kratos/commits/master/kratos/includes/oriented_bounding_box.h

I renamed (before it was obb.h) in February 2019.

https://github.com/KratosMultiphysics/Kratos/commits/master/kratos/geometries/bounding_box.h

This is from December 2019.

@pooyan-dadvand should I move the OBB to geometries?, should we unify the bounding boxes?

Reply:
@loumalouomega if you agree, this can be merged and a new issue can be opened to follow this question.

Reply:
Okay, I have not blocked